python_ta/reporters/html_reporter.py

```diff
@@ -84,21 +86,51 @@ def output_blob(self):
         # Date/time (24 hour time) format:
         # Generated: ShortDay. ShortMonth. PaddedDay LongYear, Hour:Min:Sec
         dt = str(datetime.now().strftime('%a. %b. %d %Y, %I:%M:%S %p'))
-        output_path = os.path.join(os.getcwd(), self.linter.config.pyta_output_file)
+
+        # Render the jinja template
+        rendered_template = template.render(date_time=dt, reporter=self)
+
+        # If a filepath was specified, write to the file
+        if self._output_filepath:
+            self._write_html_to_file(rendered_template)
+        else:
+            self._open_html_in_browser(rendered_template.encode('utf8'))
+
+    def _write_html_to_file(self, rendered_template):
```

Comment:
Please include type annotations (we haven't always been consistent on this, but should be for new code).

Reply:
okay, thank you for the clarification. I didn't include them originally to try and be consistent with the codebase, but I will gladly add them for future changes 👍